app.js

```diff
@@ -2,22 +2,120 @@
+          const paypalItemName = document.createElement("eager-paypal-item-name")
+          const paypalPrice = document.createElement("eager-paypal-price")
+
+          let currencySymbol
+          let times
+
+          if (options.region.currency === "GBP") {
+            currencySymbol = "£"
+          }
+          else {
+            currencySymbol = "$"
+          }
+
+          paypalItemName.innerHTML = `${attrs.name}`
+
+          paypalPrice.innerHTML = `${currencySymbol}${attrs.amount} ${options.region.currency} `
 
-  function updateElement() {
+          if (attrs.type === "subscribe") {
+            if (attrs.recurrence === 1){
+              times = "time"
+            }
+            else {
+              times = "times"
+            }
+            paypalPrice.innerHTML += `${attrs.recurrence} ${times} ${TIME_PERIOD_SYMBOLS[attrs.timePeriod]} `
+          }
+
+          if (options.region.tax && !attrs.shipping) {
+            paypalPrice.innerHTML += `<small>+ ${currencySymbol}${options.region.tax} tax </small>`
+          }
+          else if (attrs.shipping && !options.region.tax) {
+            paypalPrice.innerHTML += `<small>+ ${currencySymbol}${attrs.shipping} shipping</small>`
+          }
+          else if (attrs.shipping && options.region.tax) {
+            const shippingAndTax = attrs.shipping + options.region.tax
+
+            paypalPrice.innerHTML += `<small>+ ${currencySymbol}${shippingAndTax} shipping and tax</small>`
+          }
+
+          // TODO find production host
+          paypalButton.src = `${PAYPAL_SCRIPT_URL}?merchant=${options.merchant}`
+          paypalButton.async
+          paypalButton.setAttribute("data-button", attrs.type)
+          paypalButton.setAttribute("data-type", "form")
+          paypalButton.setAttribute("data-name", attrs.name)
+          if (attrs.type === "donate"){
+            paypalButton.setAttribute("data-amount-editable", 100)
+          }
+          else {
+            paypalButton.setAttribute("data-amount", attrs.amount)
+          }
+          paypalButton.setAttribute("data-currency", options.region.currency)
+          if (attrs.type === "buynow" || attrs.type === "cart") {
+            paypalButton.setAttribute("data-quantity-editable", 1)
+          }
+          else {
+            paypalButton.setAttribute("data-quantity", 1)
+          }
+          paypalButton.setAttribute("data-tax", options.region.tax)
+          paypalButton.setAttribute("data-shipping", attrs.shipping)
+          paypalButton.setAttribute("data-size", "small")
+          paypalButton.setAttribute("data-style", attrs.style)
+          if (attrs.type === "subscribe") {
+            paypalButton.setAttribute("data-recurrence", attrs.recurrence)
+            paypalButton.setAttribute("data-period", attrs.timePeriod)
+          }
+
+          const element = elements[i] = Eager.createElement(attrs.location, elements[i])
+
+          element.appendChild(paypalInfoWrapper)
```

Comment:
Move this to after the `infoWrapper` has appending its children to reduce the jittery render as each child is appended while the parent is mounted.